libvirt/disk_def_test.go

```diff
@@ -13,7 +13,10 @@ func init() {
 }
 
 func TestDefaultDiskMarshall(t *testing.T) {
-	b := newDefDisk()
+	b := newDefDisk(0)
+	prettyB := spew.Sdump(b)
+	t.Logf("Parsed default disk:\n%s", prettyB)
```

Comment:
i would like to propose that we don't print to much log details.

It becomes really difficult to have 3/5 tests that are printing so much info, and then to filter what is passing and what is not

Reply:
Did I added that, or did it come from the rebase?

Reply:
dunno :roll_eyes: :rofl: .